.travis/build.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash -x
+set -e
+
+CWD=$(pwd)
+export SLOW_MACHINE=1
+export PATH=$CWD/dependencies/bin:$CWD/dependencies/usr/local/bin/:/tmp/lightning/lightningd/:"$HOME"/.local/bin:"$PATH"
+export PYTEST_PAR=10
+export TEST_DEBUG=1
+export LIGHTNING_VERSION=${LIGHTNING_VERSION:-master}
+export PYTHONPATH=/tmp/lightning/contrib/pyln-client:/tmp/lightning/contrib/pyln-testing:/tmp/lightning/contrib/pylightning:$$PYTHONPATH
+
+mkdir -p dependencies/bin || true
+
+# Download bitcoind and bitcoin-cli 
+if [ ! -f dependencies/bin/bitcoind ]; then
+    wget https://bitcoin.org/bin/bitcoin-core-0.17.1/bitcoin-0.17.1-x86_64-linux-gnu.tar.gz
+    tar -xzf bitcoin-0.17.1-x86_64-linux-gnu.tar.gz
+    mv bitcoin-0.17.1/bin/* dependencies/bin
+    rm -rf bitcoin-0.17.1-x86_64-linux-gnu.tar.gz bitcoin-0.17.1
+fi
+
+pyenv global 3.7.1
```

Comment:
Well, this is supposed to run inside a travis docker image, which does have quite a bit of extra stuff. This command just pins the python version to a specific release, instead of whatever the default was.